src/modules/afterScreenshot.js

```diff
@@ -16,7 +16,8 @@ export default async function afterScreenshot(browser, options) {
 
         for (let i = 0; i < options.hide.length; i++) {
             let elements = await getElements(options.hide[i]);
-            await browser.execute(hideElements, elements, false);
+            // browser.execute cannot convert array of elements to corresponding DOM element
+            elements.forEach(async elem => await browser.execute(hideElement, elem, false));
```

Comment:
async und `forEach` vertragen sich nicht.

Hier bitte eine `for ... of` -Schleife verwenden.